scripts/copySolutionsFromStack.js

```diff
@@ -0,0 +1,42 @@
+const cheerio = require('cheerio');
+const fs = require('fs');
+const path = require('path');
+const fsEx = require('fs-extra');
+
+function main(repositoryDir) {
+    let dirContent = fs.readdirSync(repositoryDir);
+    
+    dirContent.forEach(function (dirItem){
+        item = `${repositoryDir}/${dirItem}`;
+        fileStats = fs.lstatSync(item);
+
+        if(fileStats.isDirectory()){
+            if(path.extname(item) == '.wiki'){
+                let subfolders = fs.readdirSync(item);
+                let prefix = path.basename(item,'.wiki') + '_';
+
+                subfolders.forEach(function (subfolder){
```

Comment:
You do not need to search for the folder. You can just check if a subfolder 'solutions' exists and then just take it.